### DISCUSSION TITLE
Looking for example of react-unity-webgl using Depanther's webXR

### DISCUSSION THREAD

**Discussion by DaveXR** [jeffreylanters/react-unity-webgl#410] [Help Wanted!]
I'm using both this repo and this one to enable VR mode: https://github.com/De-Panther/unity-webxr-export/

I'm getting a bit stuck on the messaging which the XR module sends out to say if VR or AR are available. Works fine in normal vanilla JS. Listening with React, I'm not getting it. Anyone else run into this?

**Comment by jeffreylanters** [comment]
Hi. A recent release introduced a feature that might help you with this implementation. See #435 for further details and discussions.

**Comment by learning-github-zgy** [comment]
can you solve it? l have the same question with you
